app/scripts/services/dimFarmingService.factory.js

```diff
@@ -190,7 +185,9 @@
           }).filter((item) => !_.isUndefined(item));
 
           self.farmItems().then(function() {
-            self.makeRoomForItems();
+            if (settings.makeRoomForItems !== false) {
```

Comment:
Again, didn't mean to push. Also, it's written that way as adding settings doesn't make them persist right away; you have to open the settings dialog to get any new setting to persist (or manually call the save() method on the settings service). Anyway, it's all still WIP.